bin/wmagent-upload-config

```diff
@@ -1,17 +1,24 @@
 #!/usr/bin/env python
 from __future__ import division, print_function
 import os
+from pprint import pformat
 from WMCore.Agent.DefaultConfig import DEFAULT_AGENT_CONFIG
 from WMCore.Configuration import loadConfigurationFile
 from WMCore.Services.ReqMgrAux.ReqMgrAux import ReqMgrAux
 
-def insertWMAgentConfig(reqmgrURL, hostName, agentConfig):
-    reqMgrAux = ReqMgrAux(reqmgrURL)
-    reqMgrAux.postWMAgentConfig(wmConfig.Agent.hostName, agentConfig)
+
+def insertWMAgentConfig(reqMgrAux, agentName, agentConfig):
+    print("Uploading agent configuration to ReqMgrAux. Data is\n%s" % pformat(agentConfig))
+    reqMgrAux.postWMAgentConfig(agentName, agentConfig)
+
+
+def removeWMAgentConfig(reqMgrAux, agentName):
+    reqMgrAux.deleteWMAgentConfig(agentName)
+
 
 if __name__ == "__main__":
     wmConfig = loadConfigurationFile(os.environ['WMAGENT_CONFIG'])
-    
-    insertWMAgentConfig(wmConfig.General.ReqMgr2ServiceURL,
-                        wmConfig.Agent.hostName, DEFAULT_AGENT_CONFIG)
-    
+    reqMgrAux = ReqMgrAux(wmConfig.General.ReqMgr2ServiceURL)
+
+    removeWMAgentConfig(reqMgrAux, wmConfig.Agent.hostName)
```

Comment:
What happens when remove fails (if document doesn't exists) will it through error or move on. (Of course, we should print message and move on)

Reply:
I wanted to ask you this question but I forgot. 
Should we raise an exception as we do here: https://github.com/dmwm/WMCore/blob/master/src/python/WMCore/ReqMgr/Service/Request.py#L492
?

Reply:
I don't think we should raise error here. but catch error and print log and move on. (Since some cases - a brand new agent won't have document initially.

Reply:
It will log the error and move on. I'm actually not sure whether we really need the `CouchError` exception class.